src/vs/workbench/browser/parts/activitybar/activitybarPart.ts

```diff
@@ -823,7 +823,6 @@ export class ActivitybarPart extends Part implements IPaneCompositeSelectorPart
 					id: cachedViewContainer.id,
 					name: cachedViewContainer.name,
 					order: cachedViewContainer.order,
-					badgeEnabled: cachedViewContainer.badgeEnabled,
 					pinned: cachedViewContainer.pinned,
```

Comment:
same comment from panelPart

Reply:
It's because the composite bar now directly interfaces with the `viewDescriptorService` which handles all the caching etc. No need to pass it along in the creation of the item

Reply:
why do we have it in the cachedpanel/viewcontainer then below?

Reply:
i see it removed now, thanks

Reply:
Where? I believe I removed it